<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>721</width>
      <height>491</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_Scrape">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>350</y>
       <width>251</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Scrape!</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>420</width>
       <height>184</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>400</width>
        <height>164</height>
       </rect>
      </property>
      <property name="firstDayOfWeek">
       <enum>Qt::Monday</enum>
      </property>
      <property name="gridVisible">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QCalendarWidget::SingleSelection</enum>
      </property>
      <property name="navigationBarVisible">
       <bool>true</bool>
      </property>
      <property name="dateEditEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>703</width>
       <height>49</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>11</y>
        <width>661</width>
        <height>27</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="pushButton_exportFolder">
       <property name="text">
        <string>Export Folder</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_expoortFolder">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>270</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_numberSelectedDays">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>153</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of selected days:</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>253</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>fitbit scraper</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>180</width>
       <height>71</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Amount of accumulated days:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_numberDays">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>80</y>
       <width>256</width>
       <height>261</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>List of  Users:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_USers"/>
    </widget>
    <zorder>frame_3</zorder>
    <zorder>pushButton_Scrape</zorder>
    <zorder>frame_2</zorder>
    <zorder>spinBox_numberDays</zorder>
    <zorder>label</zorder>
    <zorder>frame_4</zorder>
    <zorder>listWidget_USers</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>splitter_3</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
